api/desecapi/exceptions.py

```diff
@@ -25,3 +25,9 @@ def __init__(self, response=None):
 class PDNSException(APIException):
     def __init__(self, response=None):
         return super().__init__(f'pdns response code: {response.status_code}, pdns response body: {response.text}')
+
+
+class ConcurrencyException(APIException):
+    status_code = status.HTTP_429_TOO_MANY_REQUESTS
+    default_detail = 'Too many concurrent requests.'
+    default_code = 'too_many_requests'
```

Comment:
Better: `too_many_concurrent_requests`; the other value sounds like it's a result of rate limiting.

The reason is actually not that it is two many requests in the sense that could configure a limit other than 2 concurrent requests. The reason really is that a concurrency inconsistency / race condition is happening. So, you about `default_code = 'concurrency_race'`, `'concurrency_conflict`', `'concurrency_inconsistency'` or similar?

Reply:
I like `concurrency_conflict`